### DISCUSSION TITLE
Separate window on Debian

### DISCUSSION THREAD

**Discussion by massimopeca** [smplayer-dev/smplayer#611] [General]
Separate window on Debian

Hello,
I can not see the movies are with the smplayer window, because it opens only another of mpv, despite the option is disabled "Run MPlayer/mpv in a separate window".
I use Debian 11 and the pre-installed version of smplayer (20.6.0 -revision 9418).
What can I do?
![Schermata del 2022-12-08 12-39-49](https://user-images.githubusercontent.com/120101918/206440687-d2c814d5-9e76-4a34-bd53-e4b34be0dc56.png)

Thank you
![Schermata del 2022-12-08 12-47-32](https://user-images.githubusercontent.com/120101918/206440703-15be9174-ca9d-408d-b4ff-adb85443b713.png)

**Comment by smplayer-dev** [comment]
Try selecting another video output in preferences -> general -> video. If that doesn't work, try a snap or flatpak package:
https://blog.smplayer.info/how-to-fix-the-video-problem-on-wayland/
